build.sbt

```diff
@@ -5,6 +5,10 @@ enablePlugins(ZioSbtEcosystemPlugin, ZioSbtCiPlugin)
 inThisBuild(
   List(
     name              := "ZIO Redis",
+    zioVersion        := "2.0.16",
+    scala212          := "2.12.18",
+    scala213          := "2.13.11",
+    scala3            := "3.3.0",
```

Comment:
These are yet to be upgraded in the plugin, right?

Reply:
Not sure. I think I updated them recently in zio-sbt but I think you shouldn't have to wait for zio-sbt to update the Scala versions and make a new release for you to update yours